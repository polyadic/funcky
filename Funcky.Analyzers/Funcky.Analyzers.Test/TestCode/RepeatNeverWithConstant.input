using System;
using System.Linq;

namespace ConsoleApplication1
{
    class Program
    {
        private void Syntax()
        {
            const int never = 0;
            var single = Enumerable.Repeat("Hello world!", never);
        }
    }
}
