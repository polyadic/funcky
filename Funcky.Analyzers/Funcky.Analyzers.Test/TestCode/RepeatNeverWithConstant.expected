using System;
using System.Linq;

namespace ConsoleApplication1
{
    class Program
    {
        private void Syntax()
        {
            const int never = 0;
            var single = Enumerable.Empty<string>();
        }
    }
}
